content/codespaces/managing-codespaces-for-your-organization/choosing-who-owns-and-pays-for-codespaces-in-your-organization.md

```diff
@@ -54,9 +54,9 @@ A codespace is paid for by the account that owns it. The codespace owner can be
 
 If your organization owns a codespace, your organization has control over that codespace. For example, for codespaces owned by your organization, you can:
 
-- Use the [REST API](/rest/codespaces/organizations) to manage codespaces, such as stopping or deleting a codespace
-- Access audit logs to review actions related to {% data variables.product.prodname_github_codespaces %}
-- Set policies to manage constraints, such as restricting the base image or machine type that can be used in codespaces, or setting a default timeout and retention period
+- Use the [REST API](/rest/codespaces/organizations) to manage codespaces, such as stopping or deleting a codespace.
+- Access audit logs to review actions related to {% data variables.product.prodname_github_codespaces %}.
+- Set policies to manage constraints, such as restricting the base image or machine type that can be used in codespaces, or setting a default timeout and retention period.
```

Comment:
We only use periods at the end of list items when each list item is a full sentence. In this case, each list item is a continuation of the introductory sentence rather than a sentence of its own.

```suggestion
- Use the [REST API](/rest/codespaces/organizations) to manage codespaces, such as stopping or deleting a codespace
- Access audit logs to review actions related to {% data variables.product.prodname_github_codespaces %}
- Set policies to manage constraints, such as restricting the base image or machine type that can be used in codespaces, or setting a default timeout and retention period
```